coral/src/app/features/topics/overview/messages/components/TopicMessageItem.tsx

```diff
@@ -13,6 +13,25 @@ function TopicMessageItem({ offsetId, message }: Props) {
   const [expanded, setExpanded] = useState<boolean>(false);
   const panelTrigger = useId();
   const panelId = useId();
+
+  function getMessage(): React.ReactNode {
```

Comment:
Nitpick:
- maybe this function could be defined outside of the scope of `TopicMessageItem` for marginal performance benefits (and maybe legibility)
- we use `!message` a few times, but we actually want to assert that `message.length === 0` right? I kind of am irrationally attached to making boolean checks avoid type coercion when possible ^^'

Reply:
You're not wrong, but let's leave this as housekeeping 👍🏼